doc/ocpp_201_status.md

```diff
@@ -1225,58 +1224,57 @@ This document contains the status of which OCPP 2.0.1 numbered functional requir
 |-----------|--------|--------|
 | J03.FR.04 |        |        |
 
-
 ## SmartCharging - SetChargingProfile
 
 | ID        | Status | Remark                                                                                          |
 |-----------|--------|-------------------------------------------------------------------------------------------------|
 | K01.FR.01 | 🌐     | `TxProfile`s are supported.                                                                     |
 | K01.FR.02 | 🌐     |                                                                                                 |
 | K01.FR.03 | 🌐 💂  | `TxProfile`s without `transactionId`s are rejected.                                             |
-| K01.FR.04 | ✅     |                                                                                                 |
+| K01.FR.04 | 🌐     |                                                                                                 |
 | K01.FR.05 | ✅     |                                                                                                 |
-| K01.FR.06 | 🌐     |                                                                                                 |
-| K01.FR.07 | ⛽️     | Notified through the `signal_set_charging_profiles` callback.                                   |
+| K01.FR.06 | ✅     |                                                                                                 |
+| K01.FR.07 | ⛽️K08  | Notified through the `signal_set_charging_profiles` callback.                                   |
 | K01.FR.08 | 🌐     | `TxDefaultProfile`s are supported.                                                              |
 | K01.FR.09 | ✅     |                                                                                                 |
-| K01.FR.10 | ✅     |                                                                                                 |
-| K01.FR.11 |        |                                                                                                 |
-| K01.FR.12 |        |                                                                                                 |
-| K01.FR.13 |        |                                                                                                 |
+| K01.FR.10 | ⛽️K08  |                                                                                                 |
```

Comment:
There's still stuff relevant for libocpp in this one, and we enforce it in how we store the profiles

Reply:
I added a comment about the values being set during validation. I still think that it is accurate that the bulk of the work is in K08 for this though.